### ISSUE TITLE
Cryptic error message if package.json contains an error

### ISSUE THREAD

**Issue by pqvst** [vercel/vercel#893]
I accidentally forgot to remove a trailing comma in my `package.json`. When trying to deploy with a faulty `package.json` file, now returns this cryptic error message. It would be nice if that error at least mentioned `package.json`.

````
> Error! An unexpected error occurred!
  SyntaxError: Unexpected token } in JSON at position 133
    at JSON.parse (<anonymous>)
    at readJSON (/snapshot/now-cli/dist/now.js:4484:17) SyntaxError: Unexpected token } in JSON at position 133
    at JSON.parse (<anonymous>)
    at readJSON (/snapshot/now-cli/dist/now.js:4484:17)
````

**Comment by sholladay** [comment]
Apologies for the commit link spam... I kept finding more things to fix related to this. I _think_ that's all of them. :)

This is fixed in PR #899.

**Comment by Qix-** [comment]
It looks like @sholladay rose to the occasion here :) Thanks!
